test/Interpreter/SDK/Inputs/objc_implicit_inner_pointer/objc_implicit_inner_pointer.m

```diff
@@ -0,0 +1,27 @@
+#import "objc_implicit_inner_pointer.h"
+
+@implementation Foo {
+  CFTypeRef _bar;
+}
+
+- (id)init {
+  _bar = (__bridge_retained CFTypeRef)[[NSNumber alloc] initWithInt:1234567891];
```

Comment:
Ah, it's probably best not to use an NSNumber, because those can get uniqued! Maybe an NSMutableString? ("mutable" avoids uniquing there)

Reply:
\*slaps forehead\* Thanks. I was trying to avoid the tagged pointers thing but this is much simpler.